### PR TITLE
fix: country dropdown menu should not show random info

### LINKED ISSUE #5438: Country Field in Billing Information: Only show country dropdown, do not automatically fill in random info - only country
* [ ] Fix Country dropdown. It automatically shows data saved in the browsers instead of using the dropdown country option

Expected: Only show country options. Do not show other possible data saved in the browser.

Screenshot from 2020-07-30 10-25-29

### PR DESCRIPTION
**Pull request by ankingcodes**
Fixes #5438 

#### Short description of what this resolves:
Fixes the country dropdown which doesn't respond to `autocomplete="no"` in the nested `<input>`

#### Changes proposed in this pull request:

- Changed the `class` of the DropDown to `selection` since the issue is due to the lack of `autocomplete="no"` in the input tag of semantic UI library
- Removed the nested `<Input/>` since it was not used at all.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by ankingcodes** [comment]
@mariobehling @iamareebjamal - This is ready for review 👍🏻

File: app/templates/components/forms/user-payment-info-form.hbs [hunk 1]
```diff
@@ -57,15 +57,10 @@
   <div class="field">
     <label >{{t 'Country'}}</label>
     <UiDropdown
-      @class="search selection"
+      @class="selection"
```
Comment (iamareebjamal) [inline app/templates/components/forms/user-payment-info-form.hbs:60]: We want it searchable

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
We want searchable field

File: app/templates/components/forms/user-payment-info-form.hbs (hunk 1 above)
Comment (ankingcodes) [reply app/templates/components/forms/user-payment-info-form.hbs:60]: Adding `search` would use a nested `<input>` tag which doesn't have `autocomplete="off"`. Is there a way to set that ?

Comment (iamareebjamal) [reply app/templates/components/forms/user-payment-info-form.hbs:60]: Yes, see Tax Info box in Create Event page

**Pull request closed without merging**